blueprints/vendor-shim/files/vendor/__name__-shim.js

```diff
@@ -0,0 +1,9 @@
+(function() {
```

Comment:
@rwjblue that's fine.

Would `vendor/shims/moment.js` be a good auto-generated name?